contracts/stellar-axelar-gateway/src/auth.rs

```diff
@@ -39,24 +29,15 @@ pub fn initialize_auth(
 }
 
 pub fn domain_separator(env: &Env) -> BytesN<32> {
-    env.storage()
-        .instance()
-        .get(&DataKey::DomainSeparator)
-        .expect("domain_separator not found")
+    storage::domain_separator(env).expect("domain_separator not found")
```

Comment:
We should consider creating two getter versions, one that requires the key to exist and another that returns an Option. The former can panic if it's not found, and will be the more commonly used. The panic msg is ignored so we can use unwrap instead when we refactor